internal/pkg/varfile/varfile.go

```diff
@@ -217,9 +221,30 @@ func decode(filename string, src []byte, ctx *hcl.EvalContext, target *variables
 		oRange := hcl.RangeBetween(kv.Key.Range(), kv.Value.Range())
 		fixupRange(&oRange)
 
+		var path pack.ID
+		var name variables.ID
+
+		// NOTE: This implementation assumes a single element variable value,
```

Comment:
I think that's fine for now but maybe make an issue so that we could revisit later?

Reply:
Created issue #435 to capture this